include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -214,16 +215,31 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
    * \brief Publish an end effector to rviz
    * \param pose - the location to publish the marker with respect to the base frame
    * \param ee_jmg - joint model group of the end effector, e.g. "left_hand"
+   * \param ee_joint_pos - position of all active joints in the end effector
    * \param color to display the collision object with
    * \return true on success
    */
   bool publishEEMarkers(const Eigen::Affine3d& pose, const robot_model::JointModelGroup* ee_jmg,
+                        const std::vector<double>& ee_joint_pos,
+                        const rviz_visual_tools::colors& color = rviz_visual_tools::CLEAR,
```

Comment:
@simonschmeisser why did you choose CLEAR here? I just debugged why moveit_grasps didn't visualize animated grasps anymore, and it took a long time for me to realize alpha was set to 0 by default! I'm going to make the default something else...